xml/System.Security.Cryptography/ECDiffieHellmanCng.xml

```diff
@@ -1109,9 +1118,25 @@
         <ReturnType>System.Security.Cryptography.KeySizes[]</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets the key sizes, in bits, that are supported by the <see cref="KeySize"/> property setter.</summary>
```

Comment:
`<see cref` elements need to point to the full DocId, including their type prefix.